src/resource/resource.ts

```diff
@@ -176,6 +180,29 @@ export function isPodOwner(
   return podOwner === webId;
 }
 
+/**
+ *  Given an solidDataset, fetch the urls of all contained resources.
```

Comment:
Using `[[` and `]]` (or `{@link` and `}`), TypeDoc will include a link to its definition:

```suggestion
 *  Given a [[SolidDataset]], fetch the urls of all contained resources.
```